### DISCUSSION TITLE
Antarctica

### DISCUSSION THREAD

**Discussion by gavinny** [NASA-GISS/freshwater-forcing-workshop#3] [Antarctica]
This discussion is about the observations related to historical fluxes from the Antarctic ice sheets/ice shelves and peripheral glaciers.

**Comment by jlbamber** [comment]
Antarctica (AIS). Ice sheet mass balance (partitioned between SMB and grounding line discharge) is available from 1992-present. IOM estimates go back to 1979 with larger uncertainties. Prior to this it is possible to use closure of the sea level budget (Frederikse et al 2020) to provide a constraint on early 20th century grounded ice mass balance but not on ice shelf mass change. This early 20th C trend will be smooth and likely smoother than the real variability in FW flux from the ice sheet. Annually it is good to around +- 100 Gt (0.3 mm SLE) but the cumulative errors are smaller than this (see Frederikse et al 2020) or ESA SLB project.

Relatively large differences between ice shelf volume change estimates from different groups and studies. Was suggested that IMBIE could extend its work to include the ice shelves which would allow a synthesis of the different estimates for use in calculcating FW fluxes. This would be beneficial

**Reply by ben-davison** [reply]
For anomalous FW from ice shelves, we can use the Greene et al. (2022) calving-induced mass changes during 1997-2021 and thinning (anomalous melt) during 1992-2017 from Paolo et al. (2023) - more recent data may be available from Tom Slater and @AndyShepCPOM. There is a risk of overestimating the calving flux and underestimating the thinning because of changes in ice shelf area, but those should be fine as a rough approximation. Adding the mass change of the upstream basin feeding the ice shelf should also (I think) implicitly account for any slowdown in thinning owing to increases in advection (e.g. at Pine Island). My input-output approach is useful for total freshwater export, but the anomalies have large uncertainties.

Prior to 1997 observations for ice shelves are much sparser. Decadal area changes on the Peninsula are available since 1940 from Cook and Vaughan (2010). Those were converted to mass since 1990 in Slater et al. (2021), which includes the majority of area changes. Some large calving events and areas are known back to at least 1963, and more mapping may be possible every ~10 years from 1960 using Argon and Landsat (Bertie Miles has mosaics). I'm not aware of any readily-available info prior to 1960 - probably BAS and AWI have some old photos... Sediment cores suggest that the Amundsen Sea had it's current configuration prior to 1940, so one can assume steady-state with variability superimposed.

**Comment by nicojourdain** [comment]
In case this is of any use, I have [a paper in discussion](https://doi.org/10.5194/egusphere-2024-58) on the emulation of MAR's surface mass balance back to 1850 based on the CMIP6 multi-model ensemble trajectory from 1850 to present-day. See Fig. 6 and description:

> _the grounded ice sheet SMB was 110 Gt yr−1 lower in 1850–1869 than in 1995–2014 and increased slowly through the 20th century (Fig. 6a). In comparison, a combination of ice cores and simulations from another regional climate model gave ∼200 Gt yr−1 of difference between these two periods ([Thomas et al., 2017](https://doi.org/10.5194/cp-13-1491-2017))._

I guess these numbers could be used to separate {iceberg+ice-shelf} melting from the SMB signal when we consider sea level data.

**Comment by nicojourdain** [comment]
Another point, if needed: whatever data is used, myself and several colleagues in Grenoble can help building consistent iceberg melt patterns and time series from calving rates in a similar way as [Merino et al. (2016)](https://doi.org/10.1016/j.ocemod.2016.05.001), with possible improvements like dependency on the glacial sub-bassins of origin.

**Reply by jlbamber** [reply]
That would be really nice to incorporate for Greenland if possible as the iceberg FW can end up a long way from the calving front but it's all about time and effort.....But I guess you were thinking of AIS here?

**Reply by gavinny** [reply]
I think this is likely relevant for GrIS too, though likely it doesn't make as much difference. However once you've coded this into the model, why wouldn't you do it both both ice sheets?
